### PR TITLE
update-readme : Fix typos in readme file

### PR DESCRIPTION
**Pull request by MoulinaPradhan**
I found the following typos in the README.md.

### CODE REVIEW

**Review by sherrif10 — APPROVED** [review]
LGTM  cc @dkayiwa  , thanks for this critical eye

File: README.md [hunk 1]
```diff
@@ -131,9 +131,9 @@ https://wiki.openmrs.org/display/docs/OpenMRS+SDK
 
 OpenMRS has a modular architecture that allows developers to extend the OpenMRS core functionality by creating modules that can easily be added or removed to meet the needs of a specific implementation.
 
-Before creating your own module go to the [OpenMRS Module Repository](https://addons.openmrs.org/) and see if there is already a module for your specific use case. If so deploy and try it and if a functionality is missing join the developers of the module to add a feature.
+Before creating your own module go to the [OpenMRS Module Repository](https://addons.openmrs.org/) and see if there is already a module for your specific use case. If so deploy and try it and if functionality is missing join the developers of the module to add a feature.
```
Comment (HerbertYiga) [inline README.md:134]: thanks @MoulinaPradhan  we can also add a comma after if so around for example (if so,

Comment (MoulinaPradhan) [reply README.md]: I have committed the changes.

**Review by deshankoswatte — APPROVED** [review]
The fixes LGTM.

**Pull request closed without merging**